src/components/field/index.js

```diff
@@ -25,17 +25,7 @@ module.exports = (app) => {
              * @returns {Boolean} - Whether the field is valid or not.
              */
             invalidFieldValue: function() {
-                if (!this.validation) return null
-                if (!this.validation.$dirty) return null
-                // Validation for `requiredIf` depends on the state of other
-                // fields. Therefor don't use the $dirty check on this field,
-                // but go straight for the $invalid state.
-                if ('requiredIf' in this.validation) {
-                    return this.validation.$invalid
-                }
-
-                // Invalid has 3 states: true, false and null (not changed/dirty).
-                return this.validation.$error
+                return this.validationMessage !== ''
```

Comment:
I think this needs more work:
- The validation message should not be triggered directly after the component is mounted
- The login button stays disabled(not validated) when closing the popup and reopening